backends/tfhe-cuda-backend/cuda/src/pbs/programmable_bootstrap.cuh

```diff
@@ -1,8 +1,120 @@
-#include "../../include/device.h"
-#include "../../include/programmable_bootstrap.h"
-#include "../include/device.h"
-#include "programmable_bootstrap_classic.cuh"
-#include "programmable_bootstrap_multibit.cuh"
+#ifndef CUDA_PROGRAMMABLE_BOOTSTRAP_CUH
+#define CUDA_PROGRAMMABLE_BOOTSTRAP_CUH
+
+#include "device.h"
+#include "fft/bnsmfft.cuh"
+#include "programmable_bootstrap.h"
+#include "programmable_bootstrap_multibit.h"
+
+#include "cooperative_groups.h"
+
+using namespace cooperative_groups;
+namespace cg = cooperative_groups;
```

Comment:
Maybe these cooperative groups things should be placed in the pbs cg files? They are still there as well btw.